library/ws-widget/collection/src/lib/card-content-v2/card-content-v2.component.html

```diff
@@ -1625,7 +1625,7 @@ <h2 class="mat-subheading-2 flex-1 min-w-0 margin-remove-bottom" i18n>{{'cardcon
           <mat-icon>flip_to_back</mat-icon>
         </button>
         <mat-card-content class="min-top">
-          <div >
+          <div class="relative">
```

Comment:
Can we use any other class name instead of relative

Reply:
this is tailwindcss class